Microsoft.Toolkit.Uwp.SampleApp/SamplePages/CanvasPathGeometry/CanvasPathGeometryPage.xaml.cs

```diff
@@ -0,0 +1,253 @@
+                CanvasPathGeometry.CreateColor("#32d74b"),
+                CanvasPathGeometry.CreateColor("#ff9500"),
+                CanvasPathGeometry.CreateColor("#ffd60a")
+            };
+
+            var colorList = new List<string>()
+            {
+                "Transparent",
+                "Black",
+                "White",
+                "Crimson",
+                "Purple",
+                "LightBlue",
+                "LightGreen",
+                "Orange",
+                "Yellow"
+            };
+
+            this._samples = new List<string>()
+            {
+                string.Empty,
+                Sample1,
+                Sample2,
+                Sample3,
+                Sample4,
+                Sample5
+            };
+
+            var sampleList = new List<string>()
+            {
+                "None",
+                "Sample 1",
+                "Sample 2",
+                "Sample 3",
+                "Sample 4",
+                "Sample 5",
+            };
+
+            SampleList.ItemsSource = sampleList;
+            StrokeList.ItemsSource = colorList;
+            FillList.ItemsSource = colorList;
+
+            SampleList.SelectedIndex = 0;
+            StrokeThickness.Value = 1;
+            StrokeList.SelectedIndex = 1;
+            FillList.SelectedIndex = 0;
+
+            _selectionChanged = false;
+        }
+
+        private void OnParseData(object sender, RoutedEventArgs e)
+        {
+            _data = InputData.Text;
+            _isParsing = true;
+            SampleList.SelectedIndex = 0;
+            RenderCanvas.Invalidate();
+            _isParsing = false;
+        }
+
+        private void OnCanvasDraw(CanvasControl sender, CanvasDrawEventArgs args)
+        {
+            if (string.IsNullOrWhiteSpace(_data))
+            {
+                CommandsList.Text = string.Empty;
+                return;
+            }
+
+            if (_errorGeometry == null)
+            {
+                _errorGeometry = CanvasPathGeometry.CreateGeometry(sender, ErrorString);
+            }
+
+            _logger?.Clear();
+            CommandsList.Text = string.Empty;
+
+            try
+            {
+                _logger.AppendLine("// The following commands represent the CanvasPathBuilder command(s) needed");
+                _logger.AppendLine("// to create the CanvasGeometry from the specified Win2d Path Mini Language.");
+                var geometry = CanvasPathGeometry.CreateGeometry(sender, _data, _logger);
+                CommandsList.Text = _logger?.ToString() ?? string.Empty;
+
+                args.DrawingSession.FillGeometry(geometry, _fillColor);
+                args.DrawingSession.DrawGeometry(geometry, _strokeColor, _strokeThickness);
+            }
+            catch (Exception)
+            {
+                args.DrawingSession.FillGeometry(_errorGeometry, Colors.Black);
+                CommandsList.Text = "Parsing error! Invalid input data!";
```

Comment:
Not sure if I saw this pop-up in the sample, maybe I didn't corrupt the input enough?

Reply:
This is not a pop-up. Instead this text appears in the `Commands` Tab in the sample whenever an invalid input is entered.